proxy/certs/certs.go

```diff
@@ -226,6 +230,13 @@ func (s *RemoteCertSource) Local(instance string) (tls.Certificate, error) {
 	if err != nil {
 		return tls.Certificate{}, fmt.Errorf("couldn't parse ephemeral certificate for instance %q: %v", instance, err)
 	}
+	if s.EnableIAMLogin {
+		// Adjust the certificate's expiration to be the earlier of c.NotAfter
+		// or tok.Expiry.
+		if c.NotAfter.After(tok.Expiry) {
```

Comment:
nit: I feel like the stutter and `Not` in `NotAfter` makes this harder to read?
```suggestion
		if tok.Expiry.Before(c.NotAfter) {
```